File: Problem_Set_1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, i, j, k;

    do
    {
        height = get_int("Height: ");
    } while (height < 1 || height > 8);

    for (i = 1; i <= height; i++)
    {
        for (j = height; j > i; j--)
            printf(" ");

        for (k = 1; k <= i; k++)
            printf("#");

        printf("  ");

        for (k = 1; k <= i; k++)
            printf("#");

        printf("\n");
    }
}

File: Problem_Set_6/mario.py
# Use library provided cs50
from cs50 import get_int

# Ask until user inputs right integer
while True:
    integer = get_int("Height: ")
    if 0 < integer < 9:
        break
# Draw the picture
for i in range(integer):
    j = integer
    while j > i + 1:
        print(" ", end="")
        j -= 1
    j = 1
    while j <= i + 1:
        print("#", end="")
        j += 1
    print("  ", end="")
    j = 1
    while j <= i + 1:
        print("#", end="")
        j += 1
    print()